test/functional/terminal/scrollback_spec.lua

```diff
@@ -416,7 +438,11 @@ describe("'scrollback' option", function()
     -- Terminal job data is received asynchronously, may happen before the
     -- 'scrollback' option is synchronized with the internal sb_buffer.
     command('sleep 100m')
-    feed_data('for i in $(seq 1 40); do echo "line$i"; done\n')
+    if iswin() then
+      feed_data('for($i=1;$i -le 40;$i++){Write-Host \"line$i\"}\r')
```

Comment:
Try `` `n `` for newline in powershell.
Use `` `r `` for carriage return and `` `r`n `` for carriage return + newline.